src/main/java/seedu/address/storage/RoomOccupancyStorage.java

```diff
@@ -94,7 +58,7 @@ public void saveNumberOfRooms(RoomList roomList, Path fileNumOfRooms) throws IOE
      * @param fileRoomsOccupied Path to where to write the room numbers of occupied rooms
      */
     public void saveOccupiedRooms(RoomList roomList, Path fileRoomsOccupied) throws IOException {
-        FileUtil.createIfMissing(fileRoomsOccupied);
+        /*FileUtil.createIfMissing(fileRoomsOccupied);
```

Comment:
if this part is not needed, then will it be better to just delete it?

Reply:
Okay will do!